test/unittest/types/topic/CMakeLists.txt

```diff
@@ -110,6 +112,8 @@ set(TEST_EXTRA_LIBRARIES
         $<$<BOOL:${WIN32}>:iphlpapi$<SEMICOLON>Shlwapi>
     )
 
+set(TEST_EXTRA_LIBRARIES)
```

Comment:
Removing this will fix the windows compilation error

Reply:
:+1: